.gitlab-ci.yml

```diff
@@ -76,7 +74,7 @@ clean-maya:
     refs:
       - /^(v[0-9][.][0-9][.]x|master)?$/
   script:
-    - sudo minikube delete || true
+    - sudo minikube delete
```

Comment:
removing || true can cause build failure in some cases need to handle it @nsathyaseelan